apis/v1alpha1/route_types.go

```diff
@@ -25,12 +25,63 @@ type GatewayObjectReference struct {
 	// +optional
 	Namespace string `json:"namespace,omitempty"`
 	// Name is the name of the referent.
-	//
-	// +kubebuilder:validation:Required
-	// +required
 	Name string `json:"name"`
 }
 
+// RouteForwardTo defines how a Route should forward a request.
+type RouteForwardTo struct {
+	// ServiceName refers to the name of the Service to forward matched requests
+	// to. When specified, this takes the place of BackendRef. If both
+	// BackendRef and ServiceName are specified, ServiceName will be given
+	// precedence. If the referent cannot be found, controllers must set the
+	// "InvalidRoutes" status condition on any Gateway that includes this
+	// Route to true.
+	//
+	// Support: Core
+	//
+	// +optional
+	// +kubebuilder:validation:MaxLength=253
```

Comment:
// +kubebuilder:validation:MinLength=1

Reply:
Does a min length make sense for an optional field?

Reply:
It does not, my bad.